arwen/contexts/runtime.go

```diff
@@ -230,8 +242,21 @@ func (context *runtimeContext) GetInitFunction() wasmer.ExportedFunctionCallback
 	return init
 }
 
-func (context *runtimeContext) MemLoad(offset int32, length int32) ([]byte, error) {
-	memory := context.instanceContext.Memory()
+func (runtime *runtimeContext) SetAsyncCallInfo(dest []byte, value []byte, gasLimit uint64, data []byte) {
```

Comment:
as AsyncCallInfo is public this setter could get the AsyncCallInfo as parameter.

Reply:
Changed.